pysteps/utils/reprojection.py

```diff
@@ -0,0 +1,87 @@
+# -*- coding: utf-8 -*-
+
+from rasterio import Affine as A
+from rasterio.warp import reproject, Resampling
+import xarray as xr
+import numpy as np
+
+
+def reprojection(R_src, R_dst):
```

Comment:
please use variable names following the standard snake_case convention, e.g.
```suggestion
def reprojection(src_array, dst_array):
```